esp-hal/src/rmt.rs

```diff
@@ -670,12 +670,10 @@ macro_rules! declare_channels {
             const NUM_CHANNELS: usize = const { 0 $( + {$num; 1} )+ };
```

Comment:
LOL `{$num; 1}` is such a good idea

Reply:
I can't say I really like it, but it does work and I really didn't want to hardcode those values :laughing: